eng/common/TestResources/Remove-TestResources.ps1

```diff
@@ -6,6 +6,7 @@
 #Requires -Version 6.0
 #Requires -PSEdition Core
 #Requires -Modules @{ModuleName='Az.Accounts'; ModuleVersion='1.6.4'}
+#Requires -Modules @{ModuleName='Az.KeyVault'; ModuleVersion='3.4.1'}
```

Comment:
Should we put this in the helper script instead of in here so that it is clear what this dependency is from?

Reply:
I don't think this script calls into the module helper script?

Reply:
It does, but not until later. If the idea was to err early, this is how to do it. Perhaps we could source the helper script earlier, though.

Reply:
It does work and the error in 7.2 is fairly decent. You want to source it early, but after setting `$ErrorActionPreference`:

```
.: C:\Users\heaths\AppData\Local\Temp\requires\foo.ps1:5
Line |
   5 |  . "$PSScriptRoot\bar.ps1"
     |    ~~~~~~~~~~~~~~~~~~~~~~~
     | The script 'bar.ps1' cannot be run because the following modules that are specified by the "#requires" statements of the script
     | are missing: ShouldNotExist.
```